crm_claim_rma_code/__openerp__.py

```diff
@@ -35,5 +18,5 @@
         'views/crm_claim_type.xml',
     ],
     'active': True,
```

Comment:
crm_claim_rma_code/__openerp__.py:5: [C8103(manifest-deprecated-key), ] Deprecated key "active" in manifest file